### ISSUE TITLE
Populated `place=borough/neighbourhood/city_block/plot` polygon with `name=*` is not rendered

### ISSUE THREAD

**Issue by joshinils** [gravitystorm/openstreetmap-carto#4987]
A polygon way without a relation or node is not rendered with its name.

I would assume that these usually do not come with a relation that combines a way and node, I currently do not know how to research this.

| TagInfo | All     | Node    | Way     | Relation | w/ name | named % |
|---------------|---------|---------|---------|----------|---------|---------|
| borough       | 1 383   | 270     | 27      | 1 086    | -       | -       |
| neighbourhood | 465 668 | 386 509 | 57 494  | 21 665   | 458 830 | 98.53%  |
| city_block    | 62 065  | 14 789  | 46 019  | 1 257    | 48 486  | 78.12%  |
| plot          | 219 043 | 2 942   | 213 730 | 2 371    | 13 066  | 5.97%   |

**Comment by joshinils** [comment]
[neighbourhood](https://imagico.de/map/styleinfo/#style=osmcarto&section=tags&key=place&value=neighbourhood) can possibly be omitted in favor of #4986, since there is already _a_ rendering for it

**Comment by imagico** [comment]
* `place=neighbourhood` is already rendered/covered in #4986.
* `place=borough` is used 1383 times - and predominantly as an addon tags on administrative boundary relations - most of which are coming from imports: https://taginfo.openstreetmap.org/tags/place=borough#chronology - the lack of adoption in manual mapping makes this unsuitable for rendering here.
* `place=plot` is disputed regarding its suitability for mapping in OSM (both regarding the scope of the project and regarding legality of recording/distributing such information) - hence unsuitable for rendering here.
* `place=city_block` has 62k uses overall but on a global level its meaning is unclear.  Different local practices of use in different parts of the world are undocumented.  How this tag relates to `place=neighbourhood` is unclear.  Many of the name tags on these do not contain proper names but zoning IDs (which are likely often non-verifiable).

**Bottom line:** Only `place=city_block` could be considered for display IMO and so far it seems doubtful if that is a good idea.  Documentation of the practice of use of that tag would help a lot.
